woocommerce/woocommerce-admin
# add action scheduler package via composer

This PR loads Action Scheduler 3.1.0 via composer.

It removes the custom AS post store which had the priority setting code.

### Detailed test instructions:

- Install & activate Action Scheduler `master` - https://github.com/woocommerce/action-scheduler
- Edit the registered version to `3.1.1` https://github.com/woocommerce/action-scheduler/blob/master/action-scheduler.php#L39 (This works because plugins are loaded in alphabetical order.)
- Allow the migration to run
- Switch to this branch
- run `composer install`
- Add the following test snippet
```php
add_action( 'admin_menu', function() {
error_log(ActionScheduler_Versions::instance()->latest_version());	
});
```
- Load a dashboard page
- Deactivate Action Scheduler
- debug.log should have
```
[14-Jan-2020 20:07:49 UTC] 3.0.2
[14-Jan-2020 20:07:52 UTC] 3.0.2
[14-Jan-2020 20:07:52 UTC] 3.0.1
```
- Re-import your analytics data
- verify order & customer imports processed correctly

### Changelog Note:

Dev: Add Action Scheduler 3.1.0
